ui/src/app/views/homepage/component.ts

```diff
@@ -52,6 +52,7 @@ export class HomepageComponent implements OnInit, OnDestroy {
           'dolore. Mirum est notare, quam littera gothica quam nunc.',
           icon: '/assets/icons/explore.svg'
       }];
+  card: any[] = [];
```

Comment:
See above, I think it would be good to name arrays plural (`cards` in this case)